packages/instant-apps-components/src/components/instant-apps-social-share/instant-apps-social-share.test.ts

```diff
@@ -0,0 +1,300 @@
+import { expect, test, describe } from 'vitest';
+
+import '../../../dist/components/instant-apps-social-share.js';
+
+
+
+describe('test popover', async () => {
+    const social_share = document.createElement('instant-apps-social-share');
+    social_share.setAttribute('id','create-popover');
+    document.body.appendChild(social_share);
+    await new Promise(resolve => requestIdleCallback(resolve));
+
+    test('should be defaulted to popover without explicit prop', async () => {
```

Comment:
some of your tests are using `async` in the callback when it doesn't need to be used.